### DISCUSSION TITLE
Change chart type to realtime or historical based on button click and how to show chart scrollbar

### DISCUSSION THREAD

**Discussion by brijeshvpatel** [frangoteam/FUXA#992] [Q&A]
Hi,
I have following queries, if someone can help me out.

- I am not sure how to change the chart type from real time to historical and vice-versa in the same chart using button click event at the run time.
- How to show history chart and also real time data keep adding in the same chart?
- How to show chart horizontal scroll bar? And make chart autoscale off to show chart data for the specific period and then new data will show and old data will not be visible
- When you have realtime chart and you change the view and come back to again chart view, chart will get cleared and data will be rendering from the last updated data while it should be show all real time data for the specific period. This is critical.

Kind regards,
Brijesh

**Comment by unocelli** [comment]
Hi,
the feature is to be implemented #558 #615
